Sources/BraintreeApplePay/BTApplePayAnalytics.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+class BTApplePayAnalytics {
```

Comment:
We _can_ use an enum here instead like:
```
enum BTApplePayAnalytics: String {
    case paymentRequestStarted = "apple-pay:payment-request:started"
    ...
}
```

Then at the callsite we'd do `apiClient.sendAnalyticsEvent(BTApplePayAnalytics.paymentRequestFailed.rawValue)`

The advantage to a class with static properties is it's easy to bridge to our Obj-C modules vs enums (which need to be Int type in Obj-C). We would then need a separate stringValue - which may muddy up the integration pattern an hopping between Obj-C and Swift.

Reply:
Great callout! Should we leave a NEXT_MAJOR_VERSION note for when we're entirely in Swift to use the enum type?

Reply:
Good call - I'll add a note for this so we remember to re-evaluate once we are all Swift!